staging/src/k8s.io/component-helpers/scheduling/corev1/helpers.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package corev1
+
+import (
+	"encoding/json"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+// GetAvoidPodsFromNodeAnnotations scans the list of annotations and
```

Comment:
I recall we had other functions that needed to be moved. Are they coming in a followup?

Reply:
+1 yeah, many of the dependent helpers from https://github.com/kubernetes/kubernetes/issues/89930. I think the idea here was just not to overcomplicate the initial commit

Reply:
sgtm